src/Microsoft.DotNet.Build.Tasks/PackageFiles/OptionalTooling.targets

```diff
@@ -33,15 +33,15 @@
   -->
   <Target Name="GetOptionalToolingPaths">
     <PropertyGroup>
-      <OptionalToolingDir>$(ToolsDir)optional-tool-runtime\</OptionalToolingDir>
+      <OptionalToolingDir>$(ToolsDir)optional-tool-runtime</OptionalToolingDir>
```

Comment:
Why did you move the `\` from here to all of the places it's used?

Reply:
Because a quote follows the backslash further down which causes parsing errors: https://github.com/dotnet/buildtools/pull/2212/files#diff-27ba4b34f549889e007e06d384e699ecR85

Reply:
I see.